src/api/v3/feedback.py

```diff
@@ -6,6 +6,23 @@
 
 
 class FeedbackData:
+    """
+    This class generic api representation of cms feedback incl. its meta information stored in the database.
```

Comment:
I think a verb is missing in this sentence.

Reply:
True!